modules/compute/vm-instance/variables.tf

```diff
@@ -299,18 +299,36 @@ variable "bandwidth_tier" {
   }
 }
 
+variable "reservation_name" {
+  description = <<-EOT
+  If not null, use the reservation with this name.
```

Comment:
What format of names are expected?
* `name`
* `projects/<project>/reservation/<name>`
* others?